qemu/tests/cfg/multi_disk.cfg

```diff
@@ -60,6 +60,9 @@
                 stg_params = "drive_format:virtio,scsi-hd,usb2"
             usbs += " default-ehci"
             usb_type_default-ehci = usb-ehci
+            Host_RHEL.m6:
+                usbs = "default-ehci"
+                usb_type_default-ehci = ich9-usb-ehci1
```

Comment:
1. do we need to cover usb-uhci ?
2. if we just to use the default usb for rhel6, it should be more clear with following setup
Host_RHEL.m7:
             usbs += " default-ehci"
             usb_type_default-ehci = usb-ehci